composer.json

```diff
@@ -45,10 +50,64 @@
             "Cake\\Test\\": "./vendor/cakephp/cakephp/tests"
         }
     },
+    "prefer-stable": true,
```

Comment:
The `prefer-stable` could go out once all used packages are stable

Reply:
But, does it hurt?
And: Would this need an upgrade then?:
`"wyrihaximus/phpunit-class-reflection-helpers": "dev-master"`

Reply:
We can remove that, there are better ways of dealing with the issue I used it for 🤐 (learned a lot since then)

Reply:
Removed.
(I would love to know where I can read more about doing it better than using that)